docs/advanced/csidriver.md

```diff
@@ -51,25 +57,120 @@ With the kubeconfig of the Harvester cluster, you can install the third-party CS
+
+### Notable Configuration Options
+
+#### Storage Profile
+
+The CRD **StorageProfile** is the important concept for the Containerized-Data Importer (CDI). The **StorageProfile** can be used to define individual configuration for different storage providers. The following is the example for the **StorageProfile** for the LVM storage provider. We can use it to know each fields' meaning.
+
+```yaml
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: StorageProfile
+metadata:
+  name: lvm-node-1-striped
+spec:
+  claimPropertySets:
+  - accessModes:
+    - ReadWriteOnce
+    volumeMode: Block
+status:
+  claimPropertySets:
+  - accessModes:
+    - ReadWriteOnce
+    volumeMode: Block
+  cloneStrategy: snapshot
+  dataImportCronSourceFormat: pvc
+  provisioner: lvm.driver.harvesterhci.io
+  snapshotClass: lvm-snapshot
+  storageClass: lvm-node-1-striped
+```
+
+- claimPropertySets: The claimPropertySets field is used to define the accessModes and volumeMode for the StorageProfile.
+- snapshotClass: The snapshotClass field is used to define the VolumeSnapshotClass for the StorageProfile.
+- cloneStrategy: The cloneStrategy field is used to define the clone strategy for the StorageProfile.
+
+The cloneStrategy is associated with the [efficient clone](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/efficient-cloning.md).
+
+Efficient clone could be classified into three categories:
+- *CSI volume cloning* - Leverage the clone capability of the storage provider. The corresponding cloneStrategy is **csi-clone**.
+- *Smart Cloning* (snapshot-based cloning) - Leverage the snaphsot capability of the storage provider. The corresponding cloneStrategy is **snapshot**.
+- *host-assisted cloning* - Depends on directly data copy from source pod (volume) to target pod (volume). The corresponding cloneStrategy is **copy**.
 
-## Use the CSI driver
+You can define the above fields to override the default configuration showing on the status.
 
-After successfully configuring these settings, you can utilize the third-party StorageClass. You can apply the third-party StorageClass when creating an empty volume or adding a new block volume to a VM, enhancing your Harvester cluster's storage capabilities.
+### Limitations
 
-With these configurations in place, your Harvester cluster is ready to make the most of the third-party storage integration.
+- Backup support is currently limited to Longhorn V1 Data Engine volumes. Harvester is unable to create backups of volumes in external storage. 
 
-![rook-ceph-volume-external](/img/v1.2/advanced/rook-ceph-volume-external.png)
+- A known issue in the CDI prevents Harvester from converting attached PVCs to virtual machine images. Before exporting a volume in external storage, ensure that the PVC is not attached to workloads. This prevents the resulting image from getting stuck in the *Exporting* state.
```

Comment:
Suggest adding the upstream issue for reference.

Reply:
Could we use limitation instead of issue?
I will create the upstream issue, but that might need some discussion.

Reply:
Sounds good.